nodes/src/nodes/chunker/IInstance.py

```
@@ -0,0 +1,100 @@
+
+from .IGlobal import IGlobal
+
+
+class IInstance(IInstanceBase):
+    """Instance that chunks incoming documents and emits one document per chunk."""
+
+    IGlobal: IGlobal
+
+    chunkId: int = 0
+
+    def open(self, obj: Entry):
```

Ruff (0.15.10)

[warning] Unused method argument: `obj`

(ARG002)